### ISSUE TITLE
Starfield: black-rectange on lit edges are back

### ISSUE THREAD

**Issue by akarnokd** [IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT#787]
### Checklist [README]

- [X] Device is not undervolted nor overclocked
- [X] Device is using the [latest drivers](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/wiki/Intel-driver-updates-and-changelogs)
- [X] Game is not cracked, modded and use the latest patch

### Game [Required]

Starfield

### Game Platform [Required]

- [X] Steam
- [ ] Microsoft Store
- [ ] GOG Galaxy
- [ ] Ubisoft Connect
- [ ] Battle.net
- [ ] Epic Games
- [ ] Rockstar Games Launcher
- [ ] EA app
- [ ] Amazon Games app
- [ ] Other (please specify)

### Processor / Processor Number [Required]

i7 13700K

### Graphic Card [Required]

Arc A770 LE

### GPU Driver Version [Required]

31.0.101.5518

### Rendering API [Required]

- [ ] Vulkan
- [ ] OpenGL
- [X] DirectX12
- [ ] DirectX11
- [ ] DirectX10
- [ ] DirectX9

### Windows Build Number [Required]

- [X] Windows 11 23H2
- [ ] Windows 11 22H2
- [ ] Windows 11 21H2
- [ ] Windows 10 22H2
- [ ] Windows 10 21H2
- [ ] Other (Please specify)

### Intel System Support Utility report

[igcit_ssu.txt](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/files/15325096/igcit_ssu.txt)

### Description and steps to reproduce [Required]

Game version 1.11.36 just went mainstream, and with it a total of 7358 or so shaders, up from 7318 if I remember correctly.

Lit edges on characters and planets have the black rectangle graphics glitch/flickers again.

https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/assets/1269832/d2179909-e7f2-4c27-b404-9bffe9643e1a

https://www.youtube.com/watch?v=nam7wWu0bDI

### Game graphic quality [Required]

- [X] Low
- [ ] Medium
- [ ] High
- [ ] Ultra

### Game resolution [Required]

1920x1080

### Game VSync [Required]

On

### Game display mode [Required]

- [ ] Fullscreen
- [ ] Windowed
- [X] Borderless Windowed

### Detailed game settings [Required]

![Starfield_1_11_36_a](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/assets/1269832/15394e0e-0139-43a2-82f9-bf49f9f1a3be)

![Starfield_1_11_36_b](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/assets/1269832/6b423ea4-a8f5-4ace-b184-bd403b3da24a)

### Save game

Too big to upload here, even zipped.

https://drive.google.com/file/d/1734PSHCjIJ4uWxyvSYPZG32ZK4w56Cbo/view?usp=drive_link

**Comment by akarnokd** [comment]
FYI, 5522 after DDU, shaders recompiled -> same graphical glitches

**Comment by akarnokd** [comment]
FYI, 5590 after DDU, game 1.12.32.0, shaders recompiled -> same graphical glitches

**Comment by akarnokd** [comment]
FYI, 5768 after DDU, game 1.12.32.0, shaders recompiled -> same graphical glitches

**Comment by mujeeb-intel** [comment]
Hey @akarnokd do you see this issue throughout the game or on any specific mission? I remember this issue being fixed in the past but after reading your latest finding, I want to reproduce this issue again.

**Comment by akarnokd** [comment]
The issue can be observed throughout the game.  In the star system view when looking at planets. In the character menu or item preview in the inventory. Pretty much the places the original report videos show.

Also Hardware Unboxed encountered the same corruption ingame in their video a month ago: 

https://youtu.be/Y09iNxx5nFE?t=866

**Comment by mujeeb-intel** [comment]
Thanks for this. Really appreciate it. Trying to use your save file and Wow Level 144(Thats impressive).  When I try to load your save file, I get "following items are not present: Constellation.esm & OldMars.esm" When I search it in the creations menu for "constellation" I get bunch of results. Is there any specific one that you installed?

**Comment by mujeeb-intel** [comment]
Actually never mind. When I write with .esm, I can see which ones to install. all good now.

Thanks for your help

**Comment by akarnokd** [comment]
Those were the preorder bonuses that came with the game back in 2023. The save is otherwise clean.

**Comment by mujeeb-intel** [comment]
yup, was able to reproduce issue. Thank you!

**Comment by SavageWaffle** [comment]
Is this resolved?

**Comment by akarnokd** [comment]
I'll double check in about 10 hours.

**Comment by akarnokd** [comment]
Yes, still present. Game version 1.14.74.0, Driver 6449, Windows 24H2
![image](https://github.com/user-attachments/assets/45d94dc4-7824-489f-828e-d7ab23842300)

**Comment by mujeeb-intel** [comment]
Hi @akarnokd Thank you for reporting this issue. Our priority is to target the most popular games and apps to focus our efforts on providing a high quality, stable experience for the broadest set of users. We will continue to improve our software performance and compatibility throughout this year and beyond. While we can’t accommodate your request at this time, please watch the [article ](https://www.intel.com/content/www/us/en/support/articles/000099645.html)on our website for any possible changes to this situation. it''ll take some time to update but the article should be visible in a few days.
